JRFMemoryNoodler/JRFMemoryNoodler.m

```diff
@@ -132,9 +146,9 @@ static void defaultExceptionHandler (NSException *exception) {
     [[NSUserDefaults standardUserDefaults] synchronize];
 }
 
-void JRFIntentionalQuitHandler(int signal) {
-    [[NSUserDefaults standardUserDefaults] setBool:YES forKey:JRFAppDidAbortKey];
-    [[NSUserDefaults standardUserDefaults] synchronize];
+static void JRFIntentionalQuitHandler(int signal) {
+    int fd;
+    fd = creat(intentionalQuitPathname, S_IREAD | S_IWRITE);
```

Comment:
Looks like `fd` is unused here, can you either do something with it or remove it?